src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+                        updateCDPDirAndFields(cdpTextField.getText());
+                    }
+                }
+
+            }
+        });
+
+		messageLabel = new JLabel("message label");
+		messageLabel.setFont(new Font("Tahoma", Font.PLAIN, 14));
+		messageLabel.setForeground(Color.RED);
+		messageLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		messageLabel.setBounds(10, 11, 691, 32);
+		controlsPane.add(messageLabel);
+
+		checkBoxSubmitYesSamples = new JCheckBox("Submit Yes Samples");
+		checkBoxSubmitYesSamples.setSelected(true);
+		checkBoxSubmitYesSamples.setToolTipText("Only submit AC sample source");
+		checkBoxSubmitYesSamples.setBounds(33, 102, 265, 23);
+		controlsPane.add(checkBoxSubmitYesSamples);
+
+		JButton selectCDPButton = new JButton("...");
+		selectCDPButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				selectNewCDP();
+			}
+		});
+		selectCDPButton.setToolTipText("Select a different CDP");
+		selectCDPButton.setBounds(541, 56, 39, 23);
+		controlsPane.add(selectCDPButton);
+
+		JPanel LogPanel = new JPanel();
+		LogPanel.setSize(new Dimension(400, 400));
+		LogPanel.setMinimumSize(new Dimension(400, 400));
+
+		LogPanel.setLayout(new BorderLayout(0, 0));
+
+		textArea = new JTextArea();
+		textArea.setFont(new Font("Courier New", Font.PLAIN, 13));
+		textArea.setSize(new Dimension(360, 360));
+		JScrollPane scrollPane = new JScrollPane(textArea);
+		LogPanel.add(scrollPane, BorderLayout.CENTER);
+
+		JPanel panel = new JPanel();
+		panel.setPreferredSize(new Dimension(50, 50));
+		FlowLayout flowLayout_1 = (FlowLayout) panel.getLayout();
+		flowLayout_1.setHgap(45);
+		LogPanel.add(panel, BorderLayout.SOUTH);
+
+		JButton clearTextAButton = new JButton("Clear");
+		clearTextAButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				clearTextArea();
+			}
+
+		});
+		panel.add(clearTextAButton);
+
+		JButton copyButton = new JButton("Copy");
+		copyButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				ClipboardUtilities.put(textArea.getText());
+			}
+		});
+		copyButton.setToolTipText("Copy text into clipboard");
+		panel.add(copyButton);
+
+		JPanel bottomPane = new JPanel();
+		FlowLayout flowLayout = (FlowLayout) bottomPane.getLayout();
+		flowLayout.setHgap(125);
+		add(bottomPane, BorderLayout.SOUTH);
+
+		JButton submitRunButton = new JButton("Submit");
+		submitRunButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    submitSelectedJudgesSolutions();
+			}
+		});
+
+		JButton resetButton = new JButton("Reset");
+		resetButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				resetFields(true);
+			}
+		});
+		resetButton.setToolTipText("Reset back to default settings");
+		bottomPane.add(resetButton);
+		submitRunButton.setToolTipText("Submit selected sample runs");
```

Comment:
As noted elsewhere, I think the use of "sample" is potentially confusing.  I think "Submit selected Judge's solutions" would be clearer.